### PR TITLE
Remove c operators

### PR DESCRIPTION
**Pull request by Aws0mee**
Removed every C operator. I did not find and replace all, I manually checked every operator I changed so hopefully I didn't make any mistakes. I would appreciate if other could let me know if I made and incorrect changes or missed any.

### CODE REVIEW

**Comment by dvdvideo1234** [comment]
https://www.youtube.com/watch?v=755BDwzxv5c

**Comment by Aws0mee** [comment]
Just double checked all the changes and tested the code. Seems to work fine on my server. I'll convert this into an actual PR in a few days unless someone notices any issues.

**Comment by mmiller1250** [comment]
This works on my server

**Review by Vurv78 — APPROVED** [review]
Works fine in my basic tests

File: lua/entities/gmod_wire_expression2/core/table.lua [hunk 1]
```diff
@@ -1050,15 +1050,15 @@ registerCallback( "postinit", function()
 		registerOperator("idx",	id.."=ts"		, id, function(self,args)
 			local op1, op2 = args[2], args[3]
 			local rv1, rv2 = op1[1](self, op1), op2[1](self, op2)
-			if (!rv1.s[rv2] or rv1.stypes[rv2] != id) then return fixDefault(v[2]) end
+			if (!rv1.s[rv2] or rv1.stypes[rv2] ~= id) then return fixDefault(v[2]) end
```
Comment (thegrb93) [inline lua/entities/gmod_wire_expression2/core/table.lua:1053]: Missed some `!`

**Review by thegrb93 — CHANGES_REQUESTED** [review]

File: lua/entities/gmod_wire_holoemitter.lua [hunk 2]
```diff
@@ -232,32 +232,32 @@ end
...
-		if (self.Data.Pos != value) then
+		if (self.Data.Pos ~= value) then
 			self.Data.Pos = value
 			self:AddPoint()
 		end
 	else
 		-- Clear & Active
 		if (name == "Clear" or name == "Active") then
-			self:SetNWBool(name,!(value == 0 and true) or false)
+			self:SetNWBool(name,not (value == 0 and true) or false)
```
Comment (thegrb93) [inline lua/entities/gmod_wire_holoemitter.lua:257]: Just replace this garbage ternary with `value ~= 0`

File: lua/entities/gmod_wire_holoemitter.lua [hunk 3]
```diff
@@ -232,32 +232,32 @@ end
...
 		else
 			-- Other data
-			if (self.bools[name]) then value = !(value == 0 and true) or false end
+			if (self.bools[name]) then value = not (value == 0 and true) or false end
```
Comment (thegrb93) [inline lua/entities/gmod_wire_holoemitter.lua:260]: Just replace this garbage ternary with `value ~= 0`

File: lua/entities/gmod_wire_holoemitter.lua [hunk 4]
```diff
@@ -342,19 +342,19 @@ function ENT:WriteCell( Address, value )
 		self.Data.FadeTime = value
 		return true
 	elseif (Address == 9) then
-		self.Data.LineBeam = !(value == 0 and true) or false
+		self.Data.LineBeam = not (value == 0 and true) or false
```
Comment (thegrb93) [inline lua/entities/gmod_wire_holoemitter.lua:345]: Fix these too

File: lua/entities/info_wiremapinterface/init.lua [hunk 5]
```diff
@@ -55,9 +55,9 @@ local MIN_TRIGGER_TIME = 0.01 -- Minimum triggering Time for in- and outputs.
 
 -- This checks if you can give an entity wiremod abilities
 function ENT:IsWireableEntity(Entity)
-	if (!IsValid(Entity)) then return false end -- No interface for invalid entities!
-	if (IsValid(Entity._WireMapInterfaceEnt) and (Entity._WireMapInterfaceEnt ~= self)) then return false end -- Only one interface per entity!
-	if (!IsValid(Entity._WireMapInterfaceEnt) and (WireLib.HasPorts(Entity) or Entity.IsWire or Entity.Inputs or Entity.Outputs)) then return false end -- Don't destroy wiremod entites!
+	if (not IsValid(Entity)) then return false end -- No interface for invalid entities!
+	if (IsValid(Entity._WireMapInterfaceEnt) and (Entity._WireMapInterfaceEnt ~= self)) then return false end -- Only one interface per entitynot
```
Comment (thegrb93) [inline lua/entities/info_wiremapinterface/init.lua:59]: This replace is wrong

**Review by thegrb93 — APPROVED** [review]

**Pull request merged**